### DISCUSSION TITLE
Use of access-token results in http-state 403, what's wrong?

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#38] [Q&A]
Hello,
Ok, the authentication flow with SMCB does work now in TI RU (using https://id.ru.impfnachweis.info/auth/realms/ubirch-certify/protocol/openid-connect/auth), refresh of access-token also.

But using the retrieved access-token with https://api.ru.impfnachweis.info/api/certify/v2/issue results in http-state 403.
The access-token (JWT) was transfered in http-header "Authorization: Bearer *access-token*",
no client-certificate was provided.

The rest analog to Vaccination center scenario.

Whats wrong?

**Comment by dirkexe** [comment]
Please check the body of the reply. Maybe this is identical to my Issue #36? I got HTTP 403 Forbidden and "Sorry, the credential is invalid:Id not whitelisted -> Id:191872682".

**Reply by mge-medisoftware** [reply]
Thanks, but strange to say, Content-Type of response with the 403 is "application/cbor" though i used "Accept: application/pdf" in request. Body-content   seems to be empty cbor-data

**Comment by oliver-steinbrecher** [comment]
We're checking this

**Comment by nicolaslang1** [comment]
Hello
@mge-medisoftware  were you able to solve this problem?

**Reply by mge-medisoftware** [reply]
Well, that was way back.. It was solved serverside somehow i think und then we changed BSNR to a substantial BSNR because of ominous "whitelisting". Did not check RU for a while because rollout is done and certicates are successfully generated in PU since tuesday.
